components/dialog/dialog-fullscreen.js

```diff
@@ -25,68 +27,149 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 			 * Whether to render a loading-spinner and wait for state changes via AsyncContainerMixin
 			 */
 			async: { type: Boolean },
-			_hasFooterContent: { type: Boolean, attribute: false }
+			_hasFooterContent: { type: Boolean, attribute: false },
+			_icon: { type: String, attribute: false }
 		};
 	}
 
 	static get styles() {
 		return [ dialogStyles, heading3Styles, css`
 
-			.d2l-dialog-header {
-				padding-bottom: 15px;
+			.d2l-dialog-footer.d2l-footer-no-content {
+				display: none;
 			}
 
-			.d2l-dialog-header > div > d2l-button-icon {
-				flex: none;
-				margin: -4px -15px 0 15px;
+			.d2l-dialog-content-loading {
+				text-align: center;
 			}
 
-			:host([dir="rtl"]) .d2l-dialog-header > div > d2l-button-icon {
-				margin-left: -15px;
-				margin-right: 15px;
-			}
+			@media (min-width: 616px) {
+				
+				.d2l-dialog-header {
+					border-bottom: 1px solid var(--d2l-color-gypsum);
+					padding-bottom: 0.65rem;
+					padding-top: 1.15rem;
+				}
 
-			.d2l-dialog-content > div {
-				/* required to properly calculate preferred height when there are bottom
-				margins at the end of the slotted content */
-				border-bottom: 1px solid transparent;
-				box-sizing: border-box;
-				height: 100%;
-			}
+				.d2l-dialog-content > div {
+					/* required to properly calculate preferred height when there are bottom
+					margins at the end of the slotted content */
+					border-bottom: 1px solid transparent;
+					box-sizing: border-box;
+					height: 100%;
+				}
 
-			.d2l-dialog-content-loading {
-				text-align: center;
-			}
+				.d2l-dialog-header > div > d2l-button-icon {
+					flex: none;
+					margin: -4px -15px 0 15px;
+				}
 
-			dialog.d2l-dialog-outer,
-			div.d2l-dialog-outer {
-				border: none;
-				border-radius: 0;
-				box-shadow: none;
-				height: 100%;
-				max-height: initial; /* required to override Chrome native positioning */
-				max-width: initial; /* required to override Chrome native positioning */
-				top: 0;
-				width: 100%;
-			}
+				:host([dir="rtl"]) .d2l-dialog-header > div > d2l-button-icon {
+					margin-left: -15px;
+					margin-right: 15px;
+				}
+				
+				dialog.d2l-dialog-outer,
+				div.d2l-dialog-outer {
+					border-radius: 8px;
+					height: calc(100% - 3rem);
+					margin: 1.5rem;
+					max-width: 1170px;
```

Comment:
The dialog should max out at 1170px, the same as normal page content. Beyond that, center the dialog on the screen.